src/ja/common/work_machine.py

```diff
@@ -0,0 +1,33 @@
+from ja.common.message.base import Serializable
+
+
+class ResourceAllocation(Serializable):
```

Comment:
I'm not so sure about the name, but maybe something like MachineResources is better. What do you think?

Reply:
I named it ResourceAllocation because this can also be just a group of resources allocated for a job, not only the resources on a machine.